src/ai/backend/client/cli/session.py

```diff
@@ -995,23 +993,22 @@ def print_state(session_name_or_id: str, current_state_idx: int):
             print_state(session_name_or_id, current_state_idx=-1)
             async with session.listen_events(scope=scope) as response:  # AsyncSession
                 async for ev in response:
-                    match (event_name := ev.event):
-                        case SessionSuccessEvent.name:
-                            print_done(event_name)
-                            sys.exit(json.loads(ev.data).get('exitCode', 0))
-                        case SessionFailureEvent.name:
-                            print_fail(event_name)
-                            sys.exit(json.loads(ev.data).get('exitCode', 1))
-                        case KernelCancelledEvent.name:
-                            print_fail(event_name)
-                            break
+                    if ev.event == session_success:
+                        print_done(session_success)
+                        sys.exit(json.loads(ev.data).get('exitCode', 0))
+                    elif ev.event == session_failure:
+                        print_fail(session_failure)
+                        sys.exit(json.loads(ev.data).get('exitCode', 1))
+                    elif ev.event == kernel_cancelled:
+                        print_fail(kernel_cancelled)
+                        break
```

Comment:
We may use the `match case` statement. Why not?

Reply:
When comparing with any variable in `match case`, it seems that we have to wrap the constant(pattern) with string and I was not sure if it would be okay. Here is an example below:
```python
match ev.event:
    case str(session_success):
        # Do something here.
    case _:
        # Do something else here.
```
Reference: https://stackoverflow.com/questions/67525257/capture-makes-remaining-patterns-unreachable

If the session/kernel related constants are migrated into another package, then the problem would be solved.